### PR TITLE
Added support for conversions in errors

### PR DESCRIPTION
**Pull request by PierrickVoulet**
Change-Id: Ie9a2adf3da9a147554583400a4815abcf2b305cb

### CODE REVIEW

File: tests/Google/Ads/GoogleAds/Util/V2/GoogleAdsErrorsTest.php [hunk 1]
```diff
@@ -38,7 +38,6 @@ class GoogleAdsErrorsTest extends TestCase
     public function setUp()
     {
         $this->failure = $this->createGoogleAdsFailure();
-
```
Comment (fiboknacky) [inline tests/Google/Ads/GoogleAds/Util/V2/GoogleAdsErrorsTest.php]: Could you revert this change to be consistent with V1?

File: tests/Google/Ads/GoogleAds/Util/V2/GoogleAdsErrorsTest.php [hunk 2]
```diff
@@ -93,14 +92,23 @@ public function testFromFailureWithInvalidOperationIndex()
     public function testFromFailureWithMutateOperation()
     {
         $failureWithMutateOperation = $this->createGoogleAdsFailure("mutate_operations");
-
```
Comment (fiboknacky) [inline tests/Google/Ads/GoogleAds/Util/V2/GoogleAdsErrorsTest.php]: Ditto

File: tests/Google/Ads/GoogleAds/Util/V2/GoogleAdsErrorsTest.php (hunk 1 above)
Comment (PierrickVoulet) [reply tests/Google/Ads/GoogleAds/Util/V2/GoogleAdsErrorsTest.php]: Actually, I removed this line because it was not consistent with V1 (see [here](https://github.com/googleads/google-ads-php/blob/master/tests/Google/Ads/GoogleAds/Util/V1/GoogleAdsErrorsTest.php#L41))

File: tests/Google/Ads/GoogleAds/Util/V2/GoogleAdsErrorsTest.php (hunk 2 above)
Comment (PierrickVoulet) [reply tests/Google/Ads/GoogleAds/Util/V2/GoogleAdsErrorsTest.php]: Same (see [here](https://github.com/googleads/google-ads-php/blob/master/tests/Google/Ads/GoogleAds/Util/V1/GoogleAdsErrorsTest.php#L95))

Comment (fiboknacky) [reply tests/Google/Ads/GoogleAds/Util/V2/GoogleAdsErrorsTest.php]: OK. Thanks for explaining.
